{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -7,31 +7,43 @@
 from .__about__ import __version__
 
 
-################# Configuration
-config = {
-    # Add here your new settings
-    "defaults": {
-        "VERSION": __version__,
-    },
-    # Add here settings that don't have a reasonable default for all users. For
-    # instance: passwords, secret keys, etc.
-    "unique": {
-        # "SECRET_KEY": "{{ '{{' }} 24|random_string {{ '}}' }}",
-    },
-    # Danger zone! Add here values to override settings from Tutor core or other plugins.
-    "overrides": {
-        # "PLATFORM_NAME": "My platform",
-    },
-}
+################# CONFIGURATION
+hooks.Filters.CONFIG_DEFAULTS.add_items(
+    [
+        # Add here your new settings that have default values.
+        # Each new setting is a pair: (setting_name, default_value).
+        # Prefix your setting names with '{{ cookiecutter.plugin_name|upper|replace('-', '_') }}_'.
+        ("{{ cookiecutter.plugin_name|upper|replace('-', '_') }}_VERSION", __version__),
+    ]
+)
+hooks.Filters.CONFIG_UNIQUE.add_items(
+    [
+        # Add here settings that don't have a reasonable default for all users.
+        # For instance: passwords, secret keys, etc.
+        # Each new setting is a pair: (setting_name, unique_generated_value).
+        # Prefix your setting names with '{{ cookiecutter.plugin_name|upper|replace('-', '_') }}_'.
+        # For example:
+        # ("{{ cookiecutter.plugin_name|upper|replace('-', '_') }}_SECRET_KEY", "{{ '{{' }} 24|random_string {{ '}}' }}"),
+    ]
+)
+hooks.Filters.CONFIG_OVERRIDES.add_items(
+    [
+        # Danger zone! Add here values to override settings from Tutor core or other plugins.
```

Comment:
For lines `13`, `21`, and `31`:
"here" should either be at the end of the sentence, or use a command with a colon, i.e. "Add here:"

Reply:
👍🏻